internal/controllers/reconciliation/symphony_test.go

```diff
@@ -74,9 +78,40 @@ func TestSymphonyIntegration(t *testing.T) {
 	symph.Spec.Variations = []apiv1.Variation{{Synthesizer: apiv1.SynthesizerRef{Name: syn.Name}}}
 	require.NoError(t, upstream.Create(ctx, symph))
 
+	symph2 := &apiv1.Symphony{}
```

Comment:
nit: would be nice to have a comment explaining what this resource is intended to cover. Or maybe a dedicated test case for testing namespace isolation.

Reply:
Added a comment and a TODO to move this to a separate scenario.